Improve "Getting started with box" communication (#140)
liaralabs/docs.swizzin.ltd: docs/getting-started/box-basics.md

Syntax:
```bash
box
```

## help
This function provides basic information and usage examples for all availible box commands.

Syntax:
```bash
box help
```

## install
This function installs applications on the server. If you need help with specific commands, please see the `Applications` section on the left side of the website. You can specify multiple applications to install at the same time.

Syntax:
```bash
box install rtorrent deluge
```

## remove
This function removes applications from the server. If you need help with specific commands, please see the `Applications` section on the left side of the website. You can specify multiple applications to remove at once.

Syntax:
```bash
box remove rtorrent deluge
```

## update
This function updates to the latest swizzin release. This typically fixes issues with the install procress. It is often a requirement for upgrading applications or fixing broken aspects of the seedbox solution.

Syntax:
```bash
box update
```

## upgrade

> This function upgrades a specific application to a newer version. It also recompiles applications without versions to install the latest fixes. Only some applications have upgrade scripts and their name is required to upgrade them. You can view the specialized list on GitHub. https://github.com/swizzin/swizzin/tree/master/scripts/upgrade

Reviewer: ```suggestion
This function upgrades a specific application to a newer version. It can also recompile applications without versions to install the latest fixes. Only some applications have upgrade scripts and their name is required to upgrade them. You can view the specialized list on GitHub. https://github.com/swizzin/swizzin/tree/master/scripts/upgrade
```

Syntax:
```bash
box upgrade nginx
```

## adduser

> This function adds a secondary user to your server. Please note, only the primary user has access to a significant portion of applications.

Reviewer: ```suggestion
This function adds a additional users to your server. Please note, only the primary user has access to a significant portion of applications.
```

Syntax:
```bash
box adduser faithfulfriend
```

## deluser
This function removes a user from your server. Please proceed with caution because all of the user data will be destroyed.

Syntax:
```bash
box deluser exgirlfriend
```

## chpasswd

> This function changes a user password. `chpasswd` will change the password for SSH, FTP, HTTP and Deluge. Please specify `chpasswd` followed by the username you wish to change the password.

Reviewer: ```suggestion
This function changes a user password. `chpasswd` will change the password for SSH, FTP, HTTP, Deluge, and most if-not-all applications. Please specify `chpasswd` followed by the username you wish to change the password.
```
